.github/workflows/ci.yml

```diff
@@ -27,17 +27,13 @@ jobs:
         with:
           driver: docker
 
-      - name: Build Docker image
-        uses: docker/build-push-action@v5
+      - uses: actions/setup-go@v4
         with:
-          context: ./tools
-          file: ./tools/Dockerfile
-          tags: "cometbft/cometbft-db-testing:latest"
-          load: true
+          go-version: '1.22'
+          cache: false
```

Comment:
why cache is false?

Reply:
caching tends to cause issues and this is very fast anyhow